e2e/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
We initiate pushes and submit pull requests targeting the `develop` branch. Also, we don't have a `master` branch. Can you please review `master` and add `develop`?